tpp-app/src/settings/SettingsService.js

```diff
@@ -0,0 +1,116 @@
+import AsyncStorage from '@react-native-async-storage/async-storage';
+import { TRACK_SYMPTOMS, REMINDERS } from './utils/constants.js'
+
+    /**
+     * Clears all of the user's account data
+     * @returns a promise resolving when the clear operation is complete
+     */
+
+    export const DELETEAccountData = async () => new Promise( async (resolve, reject) => {
+        try {
+            await AsyncStorage.clear().then(() => {
+                console.log("Deleted user account data");
+                resolve(); 
+            })
+            
+        } catch (e) {
+            console.log(`DELETEAccountData error: ${JSON.stringify(e)}`);
+            reject();
+        }
+    })
+
+    /**
+     * Returns an array of a boolean representing if a symptom is currently tracked
+     * @returns a promise of an array containing booleans representing whether to track each symptom
+     */
+    export const GETAllTrackingPreferences = async () => new Promise (async (resolve, reject) => {
+        try {
+             await AsyncStorage.multiGet([
+                TRACK_SYMPTOMS.FLOW,
+                TRACK_SYMPTOMS.MOOD,
+                TRACK_SYMPTOMS.SLEEP,
+                TRACK_SYMPTOMS.CRAMPS,
+                TRACK_SYMPTOMS.EXERCISE
+             ]).then(() => {
+                 console.log("Got All Tracking Preferences");
+                 resolve();
+             }).catch((e) => {
+                console.log(JSON.stringify(e));
+                reject('Unable to getItem');
+             });
+        } catch (e) {
+            console.log(`GETAllTrackingPreferences error: ${JSON.stringify(e)}`)
+            reject();
+        }
+    })
+
+
+    /**
+     * Posts any changes in symptom tracking preferences 
+     * @param flow boolean representing whether to track flow
```

Comment:
![image](https://user-images.githubusercontent.com/22108651/157460283-de7184f5-3104-4865-afc5-e5cade239160.png)

If u put the type like `{boolean}` it'll be parsed by JSDoc compliant stuff. Means nice type annotation on method signatures & on autocompletion if ur editor supports JSDoc, which I think is usually the case unless ur using straight vim lol